admin/bin/validate-gpg-key.sh

```diff
@@ -58,7 +69,16 @@ printf "Validating fingerprint and public key key match...\n"
 printf "\t Public key: %s\n" "${key_location}"
 printf "\t Fingerprint: %s\n" "${fingerprint}"
 
-gpg2 --fingerprint "$fingerprint"
+gpg2 --fingerprint "$fingerprint" || report_error $KEY_MISMATCH
```

Comment:
This returns positive for cases where the supplied fingerprint is truncated by 1 char.